lib/drivers/medtronic600Driver.js

```diff
@@ -0,0 +1,1221 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2017, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+/* global chrome */
+
+const _ = require('lodash');
+const async = require('async');
+const sundial = require('sundial');
+const crypto = require('crypto');
+const crcCalculator = require('../crc.js');
+
+const isBrowser = typeof window !== 'undefined';
+// eslint-disable-next-line no-console
+const debug = isBrowser ? require('../bows')('Medtronic600Driver') : console.log;
+
+class ExtendableError extends Error {
```

Comment:
It's so that when an error is thrown, the stack trace shows the correct subclass name (otherwise it just shows `Error`). See http://stackoverflow.com/questions/31089801/extending-error-in-javascript-with-es6-syntax